app/app.js

```diff
@@ -16,6 +16,7 @@ import withParams from './containers/withParams'
 import cache from './utils/cache'
 import { parseUrl } from './utils/utils'
 import fetcher from 'utils/fetcher'
+import Welcome from './containers/WelcomeContainer'
```

Comment:
you'd still want this to be called `WelcomeContainer` to match the others (doesn't really matter but for the sake of convention).